release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -222,6 +222,15 @@ With this release, you can access the NMstate Operator and resources such as the
 
 IPsec is now supported on the IBM Cloud platform for clusters that use the OVN-Kubernetes network plugin, which is the default in {product-title} 4.14. For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
 
+[id="ocp-4-14-networking-ovn-kubernetes-ipsec-support-for-external-traffic"]
+==== OVN-Kubernetes network plugin support for IPsec encryption of external traffic
+
+IPsec now supports encryption of external traffic, also known as north-south traffic. IPsec already supports encryption of network traffic between pods, known as east-west traffic.
```

Comment:
the two can operate togther/in conjunction to provide full in-transit encryption solution for Openshift clusters